docs/sdk/server-side-sdks/java-local/java-local-gettingstarted.md

```diff
@@ -17,10 +17,10 @@ import com.devcycle.sdk.server.local.api.DVCLocalClient;
 
 public class MyClass {
     
-    private DVCLocalClient dvcLocalClient;
+    private DVCLocalClient client;
     
     public MyClass() {
-        dvcLocalClient = new DVCLocalClient("<DVC_SERVER_SDK_KEY>");
+        client = new DVCLocalClient("<DVC_SERVER_SDK_KEY>");
```

Comment:
We should set a best practice of sourcing this from an env var even in our docs.